index.js

```diff
@@ -1,39 +1,52 @@
 import process from 'node:process';
-import path from 'node:path';
-import url from 'node:url';
+import {resolve, join, delimiter} from 'node:path';
```

Comment:
I prefer to import `path` as a default import `join` and `resolve` are ambiguous without any context.